src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -87,3 +87,8 @@ export const isExternalSource = source => source && includes(['External', 'exter
 export const preventFormSubmit = event => event.target.type !== 'textarea'
     && event.which === KEY_CODE_FOR_ENTER
     && event.preventDefault();
+export const compare = (firstConcept, nextConcept) => {
```

Comment:
Looking at just the method name, what is this comparing?

Reply:
Noted. This has been addressed.